site_scons/site_tools/roc/config.py

```diff
@@ -290,6 +295,67 @@ def FindConfigGuess(context):
     context.Result('not found')
     return False
 
+def FindPkgConfigPath(context):
+    env = context.env
+
+    context.Message("Searching PKG_CONFIG_PATH...")
+
+    if env.HasArg('PKG_CONFIG_PATH'):
+        context.Result(env['PKG_CONFIG_PATH'])
+        return True
+
+    if env.HasArg('PKG_CONFIG'):
+        pkg_config = env['PKG_CONFIG']
+    elif env.Which('pkg-config'):
+        pkg_config = 'pkg-config'
+    else:
+        context.Result('not found')
+        return False
```

Comment:
If `pkg-config` tool wasn't found, we can still fallback to libdir/pkgconfig below instead of failing.